### ISSUE TITLE
Allow to save modified colors as a new color scheme.

### ISSUE THREAD

**Issue by robert-andrzejuk** [WinMerge/winmerge#1180]
I have a heavily modified "Solarized Dark" color scheme.

On the Options (Colors -> Color Schemes) I would like to press a Save button to save the settings under my own customized name in a file named different than the scheme name.

Hope this improvement is possible.

**Comment by CennoxX** [comment]
Workaround: copy and rename the file WinMerge\ColorSchemes\Solarized Dark.ini

**Comment by sdottaka** [comment]
This feature has been implemented in PR #2908.

**Comment by dcog989** [comment]
It seems this feature has introduced creation of "~\Documents\WinMerge" - even if I do not save the scheme there?

I do not like apps that create folders in *my* documents folder without permission. Can this 'feature' be removed?

Much better to open the user's 'Home' and let them choose the destination?

**Comment by sdottaka** [comment]
Currently, the Documents/WinMerge/ColorSchemes folder is not created unless you save a color scheme. We chose this location because Documents/WinMerge/Filters is already used.

It might be useful to allow users to change the location of Documents/WinMerge.

**Comment by dcog989** [comment]
No, that folder is created as soon as you click 'Save current scheme...' 

I've never used filters, so maybe I deleted that folder at some point in the past. I think it would make more sense to have those saved to "~\AppData\Roaming\WinMerge" so the user can backup / restore everything in one place?

**Comment by sdottaka** [comment]
The Documents/WinMerge/ColorSchemes folder is created when you click “Save current scheme…”, because color scheme files must be stored there in order to be detected and listed in the scheme selector.

Allowing users to choose “~\AppData\Roaming\WinMerge” as an alternative location sounds like a good idea.
